CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [0.5.26] — TBD
+### Fixed
+- Fix ordering of tables to be top to bottom ([#336](https://github.com/jsvine/pdfplumber/issues/336))
```

Comment:
Just to be clear, let's say: "Fixed `TableFinder` to sort extracted tables, ordering them from top to bottom"